server/integration/endpoint/src/main/java/org/infinispan/server/endpoint/subsystem/ProtocolServerConnectorStartHandler.java

```diff
@@ -0,0 +1,24 @@
+package org.infinispan.server.endpoint.subsystem;
+
+import org.infinispan.server.core.AbstractProtocolServer;
+import org.infinispan.server.core.configuration.ProtocolServerConfiguration;
+import org.jboss.as.controller.OperationContext;
+import org.jboss.as.controller.OperationFailedException;
+import org.jboss.as.controller.OperationStepHandler;
+import org.jboss.dmr.ModelNode;
+import org.jboss.msc.service.ServiceController;
+
+public class ProtocolServerConnectorStartHandler implements OperationStepHandler {
```

Comment:
just curious: you could have used a single class to handle the start and stop connector right? or it wouldn't work?
It seems the only difference is the method invoked :)

Reply:
I've collapsed the classes into one